### ISSUE TITLE
Enhancement suggestion: user control over js polling interval

### ISSUE THREAD

**Issue by sbulen** [SimpleMachines/SMF#6477]
#### Description
This value appears to be hard-coded:
https://github.com/SimpleMachines/SMF2.1/blob/3f33b9e4507a0b448a65de93773989305ce99a78/Themes/default/scripts/alerts.js#L1

It is a pretty critical setting that governs how often user browsers ping the forum for certain updates, e.g., notifications.  

I suspect very busy forums may want/need to throttle this. 

The only way I have found to do so at the moment is to update the source code.

Not urgent, but seems like a nice-to-have...

Note: It's in ms, so every browser pings the forum every 10 sec.

**Comment by MissAllSunday** [comment]
I agree this can be turn into a setting to control hitting the server that often but on the otherside, users who aren't that tech savvy may be inclined to use the same setting to create the false impression of "real time" notifications putting their server and quite possible other users (shared hosting) at risk of getting temp blocked by their hosting provider.

Perhaps a setting with no UI? not bullet-proof but at least might prevent a big chunk of users from changing it.

We can also provide a hard-coded min value, to prevent users from setting it to 1 for example.

**Comment by Oldiesmann** [comment]
Minimum value and/or a warning about setting it too low should be good.

**Comment by sbulen** [comment]
I am going to close this out.  I was concerned about how having many users online might cause CPU issues.

But this is not the case.  I think we're good.
